### DISCUSSION TITLE
How to get an an older version of volar?

### DISCUSSION THREAD

**Discussion by zanzara** [vuejs/language-tools#1657] [Q&A]
It is not possible to me,  to get "F12 go to definitions/references"  work with  latest version 0.39.2
On another notebook, I got the same VSCode version with volar 0.30.1 and it works there.
(All other extensions are the same)

Now i try to get somehow V.0.30.1  for Win64 platform and I don't know how.
any ideas? 
Thx.

**Comment by IIFelix** [comment]
<img width="611" alt="volar" src="https://user-images.githubusercontent.com/56734789/181990421-0c4164bd-1d02-49c3-b12f-4058ecd756ae.png">

you can click "install another version" in the vscode extension panel. It will take some time to load all available versions but after that you can select your version of choice from a list.

**Reply by zanzara** [reply]
Thanks, but it shows me nothing to install there.
Even no reaction if I hit this. :cry:

**Reply by johnsoncodehk** [reply]
@zanzara See https://github.com/johnsoncodehk/volar/issues/1192#issuecomment-1124907892.

A better way is open a bug report and attach minimal reproduction, so I can investigation with it.

**Reply by kingyue737** [reply]
> Thanks, but it shows me nothing to install there. Even no reaction if I hit this. 😢

Did you wait for the menu to popup? It is a bit slow. It takes ~10 seconds to show in my pc

![image](https://user-images.githubusercontent.com/40021217/182292253-d167d489-243f-4ca5-8f7a-d79e06034f85.png)

**Reply by zanzara** [reply]
Yes, of course: 30sec roundabout.
Nice list you show me. That's what I'd expected.

**Reply by pdob-git** [reply]
I have the same issues. I cannot install Volar 0.40
on Vscode 1.62.3
Is there somewhere list o compatible vscode version for patricular Volar version?
